pkg/api/admin/api/types.go

```diff
@@ -0,0 +1,233 @@
+package api
+
+const (
+	// APIVersion is the version of this API
+	APIVersion = "admin"
+)
+
+// OpenShiftManagedCluster complies with the ARM model of resource definition in
+// a JSON template.
+type OpenShiftManagedCluster struct {
```

Comment:
wow there is now going to be 4 versions of this struct in our code base.

Reply:
Soon to be brought down to 3 - https://github.com/openshift/openshift-azure/pull/793

But of course long-term we are going to have more, every version of the external apis is going to be its own type.